src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -319,6 +323,24 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
     }
   }
 
+  it should "persist WorkflowFailures for each workflow that timeout when submitting to Cromwell" in withWorkspaceServiceMockTimeoutExecution { mockExecSvc => workspaceService =>
+    val submissionRq = SubmissionRequest("dsde", "GoodMethodConfig", "Pair", "pair1", Some("this.case"))
+    val rqComplete = Await.result(workspaceService.createSubmission(testData.wsName, submissionRq), Duration.Inf).asInstanceOf[RequestComplete[(StatusCode, SubmissionReport)]]
+    val (status, newSubmission) = rqComplete.response
+    assertResult(StatusCodes.Created) {
+      status
+    }
+
+    val submissionStatusRq = Await.result(workspaceService.getSubmissionStatus(testData.wsName, newSubmission.submissionId), Duration.Inf).asInstanceOf[RequestComplete[(StatusCode, SubmissionStatusResponse)]]
+    val (submissionStatus, submissionStatusResponse) = submissionStatusRq.response
+    assertResult(StatusCodes.OK) {
+      submissionStatus
+    }
+
+    assert(submissionStatusResponse.notstarted.size == 1)
+    assert(submissionStatusResponse.workflows.size == 0)
+  }
+
```

Comment:
can you add a test here to check that you correctly concat normal-failures and timeout-failures together